app/src/main/res/values-sr/strings.xml

```diff
@@ -0,0 +1,138 @@
+    <string name="hide_actionbar">Sakri meni sa alatima</string>
+    <string name="resume_scroll">Nastavi skrolovanje</string>
+    <string name="uploading_image">Slika se šalje</string>
+    <string name="settings">Podešavanja</string>
+    <string name="snackbar_message_copied">Poruka kopirana</string>
+    <string name="snackbar_error_copied">Informacije o grešci kopirane</string>
+    <string name="notification_stop">Stani</string>
+    <string name="notification_title">FeelsDankMan</string>
+    <string name="notification_message">DankChat radi u pozadini</string>
+    <string name="emote_menu_hint">Otvori meni sa emotovima</string>
+    <string name="login_title">Uloguj se na Twitch.tv</string>
+    <string name="hint_connected">Počni ćaskanje</string>
+    <string name="hint_disconnected">Veza nije uspostavljena</string>
+    <string name="hint_not_logged_int">Niste prijavljeni</string>
+    <string name="notification_new_mentions">Neko te je spomenuo</string>
+    <string name="notification_mention">%1$s te je spomenuo u #%2$s</string>
+    <string name="notification_notify_mention">Spomenut si u #%1$s</string>
+    <string name="snackbar_login">Prijavljen kao %1$s</string>
+    <string name="snackbar_login_failed">Neuspešno prijavljivanje</string>
+    <string name="snackbar_image_uploaded">Kopirano: %1$</string>
+    <string name="snackbar_upload_failed">Greška prilkom slanja</string>
+    <string name="snackbar_upload_failed_cause">Greška prilikom slanja: %1$</string>
+    <string name="snackbar_retry">Pokušaj ponovo</string>
+    <string name="snackbar_data_load_failed_cause">Učitavanje podataka nije uspelo: %1$s</string>
+    <string name="add_channel_hint">Naziv kanala</string>
+    <string name="emote_menu_tab_subs">Pretplatnici</string>
+    <string name="emote_menu_tab_channel">Kanal</string>
+    <string name="emote_menu_tab_global">Globalno</string>
+    <string name="system_message_connected">Povezan</string>
+    <string name="system_message_disconnected">Veza prekiuta</string>
+    <string name="system_message_no_history">Ne vidite prethodne poruke jer ste onemogućili integraciju nedavnih poruka.</string>
+    <string name="timed_out_message">&lt; Poruka obrisana ></string>
+    <string name="multi_entry_header_regex">Regex</string>
+    <string name="multi_entry_header_match_user">Spoji korisnika</string>
+    <string name="multi_entry_add_entry">Dodajte unos</string>
+    <string name="multi_entry_remove_content_description">Izbrišite unos</string>
+    <string name="multi_entry_hint">Unos</string>
+    <string name="login_expired">Prijava istekla! Prijavite se ponovo.</string>
+    <string name="message_history_disclaimer_title">Odricanje odgovornosti za istoriju poruka</string>
+    <string name="anon_connection_disclaimer_message">Zbog nedavnih promena na zvaničnom Twitch API, nekim podacima se može pristupati samo kada ste prijavljeni. Ovo uključuje podršku za twitch značke i third party emotove</string>
```

Comment:
If you look at the original strings.xml some of the long strings have \n's inbetween for formatting and the editor doesn't show them, please make sure to add them at the correct positions